app/views/books/_taginput.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="form-group">
+  <%= label_tag plural, t("default.#{singular}.name.other"), class: "col-md-3 control-label" %>
+  <div class="col-md-6">
+    <% val = nil %>
+    <% list.each do |value| %>
+        <% val = (val || "") + value[:name] + ',' %> 
+    <% end %>
+    <%= text_field_tag "#{plural}[names]", val,
+     {id: "#{plural}-input", 'data-role' => "tagsinput",
+      'data-tag-class' => "label label-primary", class: "form-control"} %>
```

Comment:
this is a lot of code in a view. What are you trying to do here?

Reply:
to assemble authors names in a format like this `name-1, name-2`  because this format which 'tag-input' needs
ex:
`<input name="tags" id="tags-input" data-role="tagsinput" data-tag-class="label label-primary" class="form-control" value="Amsterdam,Washington,Sydney,Beijing,Cairo" />`

Reply:
usually add this type of code in application_helper or create book_helper and just call the function in the view

Reply:
ok